packages/react-reconciler/src/ReactFiberExpirationTime.js

```diff
@@ -7,12 +7,14 @@
  * @flow
  */
 
+import MAX_SIGNED_32_BIT_INT from './maxSigned32BitInt';
+
 // TODO: Use an opaque type once ESLint et al support the syntax
 export type ExpirationTime = number;
 
 export const NoWork = 0;
 export const Sync = 1;
-export const Never = 2147483647; // Max int32: Math.pow(2, 31) - 1
+export const Never = MAX_SIGNED_32_BIT_INT;
```

Comment:
I think we might actually want this to be one bit less since V8 stores signed 31-bit integers as a special path.